base/server/src/main/java/com/netscape/cmscore/apps/CMSEngine.java

```diff
@@ -392,6 +392,7 @@ public void initPasswordStore() throws Exception {
 
         boolean skipPublishingCheck = config.getBoolean("cms.password.ignore.publishing.failure", true);
         String pwList = config.getString("cms.passwordlist", "internaldb,replicationdb");
+       boolean skipLdapConnTest = config.getBoolean("cms.password.skip.ldap.conn.test",false);
```

Comment:
Looks good as long as it works.  the only suggestion I have is to have "skip.ldap.conn.test" as a single word, e.g.
skip_ldap_conn_test or skipLdapConnTest